### PR TITLE
Busser dev

### PR DESCRIPTION
**Pull request by gauravj09**

### CODE REVIEW

File: backend_models/busser.php [hunk 1]
```diff
@@ -1,14 +1,40 @@
...
+			$this->EID = $EID;
+			$this->isOccupied = false;
+			$this->database = new dbAPI;
+		}
+
+		function getDirtyTables()
+		{
+			$unclean_tables = $this->database->get_unclean_tables_list();
+
+			if($unclean_tables)
```
Comment (Ajinkya-Nawarkar) [inline backend_models/busser.php]: possibly changing this to sizeof($unclean_tables) > 0 will increase safety

Comment (gauravj09) [reply backend_models/busser.php]: Okay, I will update it. 

> possibly changing this to sizeof($unclean_tables) > 0 will increase safety

Okay, I will update it.

Comment (gauravj09) [reply backend_models/busser.php]: @Ajinkya-Nawarkar I had some trouble merging the local changes in Busser_dev to remote changes in GitHub. Just want to make sure this branch has not affected any other backend models.

Comment (Ajinkya-Nawarkar) [reply backend_models/busser.php]: What's the problem?

Comment (gauravj09) [reply backend_models/busser.php]: I had a 'failed to push' error when I tried updating Busser_dev. It said there were some conflicts between busser_dev on my local machine and busser_dev remote. After I pulled and merged busser_dev from remote and merged it with local busser_dev and pushed it again it got fixed. Just want to make sure that it didn't affect anything else.

Comment (gauravj09) [reply backend_models/busser.php]: I didn't see any errors but I want to make sure I didn't miss anything

**Pull request merged**